open-sauced/app
# fix: removed connection tab on user profile page

fix: #3141

## Description
This PR removes the "Connections" Tab from the user's profile page (Was unsure if this is considered a refactor or fix)


## Related Tickets & Documents


## Mobile & Desktop Screenshots/Recordings


I do not have a pro account with OpenSauced, so I was unable to view the "Connections" Tab on my profile page. See below:
<img width="1367" alt="Screenshot 2024-04-25 at 10 42 37 AM" src="https://github.com/open-sauced/app/assets/63355222/ef72b2c4-1365-428f-b592-5d44e8b98d43">

I followed the directions of the file to edit and the file to remove. Once the changes were made, I ensured the profile page was still populating and here is the screenshot:
<img width="1372" alt="LocallyRunningAfterChanges" src="https://github.com/open-sauced/app/assets/63355222/d184fa9d-1f6f-45b0-baec-a9b011081156">


## Steps to QA

1. From OpenSauced homepage -> click on profile picture on top right corner -> from the dropdown menu, select {username}
2. Once on Profile Page, check the tabs on the right. They should read "Highlights", "Contributions", "Recommendations"
3. "Connections" tab should be gone on the Pro Accounts. Free tiers will not see this tab populated. (I found this in the docs, but now am unable to find where I read this. I can keep searching if needed. Especially if documentation needs to be edited)
4. I did notice in the repo that lines 114-117 were showing an error. I did not touch these lines since it was not in the instructions provided. But this error does not break the page. (https://github.com/open-sauced/app/blob/18d3b15f12a993d5bc8ce0061cd4a5520b72e846/components/organisms/ContributorProfileTab/contributor-profile-tab.tsx)



## Tier (staff will fill in)


## [optional] Are there any post-deployment tasks we need to perform?



## [optional] What gif best describes this PR or how it makes you feel?

<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExanRqZXQwZzE0YTMwZnRuYmJ2MHZ6ZTk2NG9kZ2c3MWRkcnc0YXNzdSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/5J07pHrJ2lKhi/giphy.gif" />

* Hoping I followed the directions correctly
